### PR TITLE
entity

### PR DESCRIPTION
**Pull request by Gekata**
i hope its good

### CODE REVIEW

File: src/main/java/shop/entity/Order.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package shop.entity;
+
+public class Order {
+    private String id;
+    private String userid;
+    private String status;
+    private String list;
```
Comment (oleskafeska) [inline src/main/java/shop/entity/Order.java]: private List<Product> productList;

File: src/main/java/shop/entity/Product.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package shop.entity;
+
+public class Product {
+    private String id;
+    private String name;
+    private String price;
```
Comment (oleskafeska) [inline src/main/java/shop/entity/Product.java]: private double price;

File: src/main/java/shop/entity/User.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
+package shop.entity;
+
+public class User {
+    private String id;
+    private String name;
+    private String email;
+    private String PhoneNumber;
```
Comment (oleskafeska) [inline src/main/java/shop/entity/User.java]: PhoneNumber -> phoneNumber

**Review by Gekata — COMMENTED** [review]
i made it again

**Review by oleskafeska — APPROVED** [review]

**Pull request merged**